themes/default/checker/checker_results.tmpl.php

```diff
@@ -62,10 +62,10 @@
 	
 	echo '<form method="post" action="'.$_SERVER['PHP_SELF'].'">'."\n\r";
 	echo '<input type="hidden" name="jsessionid" value="'.$sessionID.'" />'."\n\r";
-	echo '<input type="hidden" name="uri" value="'.$addslashes($_POST["uri"]).'" />'."\n\r";
+	echo '<input type="hidden" name="uri" value="'.filter_var($_POST["uri"], FILTER_VALIDATE_URL).'" />'."\n\r";
 	echo '<input type="hidden" name="output" value="html" />'."\n\r";
 	echo '<input type="hidden" name="validate_uri" value="1" />'."\n\r";
-	echo '<input type="hidden" name="rpt_format" value="'.$addslashes($_POST['rpt_format']).'" />'."\n\r";
+	echo '<input type="hidden" name="rpt_format" value="'.addslashes($_POST['rpt_format']).'" />'."\n\r";
```

Comment:
To sanitize html outputs, shall we use things like `htmlspecialchars($_POST['rpt_format']), ENT_QUOTES, 'utf-8')`?

Reply:
I think that will be better